packages/alert/src/close-icon.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const CloseIconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+  pointer-events: all;
+  box-sizing: content-box;
+  svg {
+    color: ${_ => _.theme.colors.text.muted};
+    padding: 0 ${_ => _.theme.spacing.xs};
+  }
+  &:hover {
+    cursor: pointer;
+    svg {
+      color: ${_ => _.theme.colors.text.default};
+    }
+  }
+`
+
+const CloseIconFC: React.FC<{closeOnClick?: () => void}> = ({
```

Comment:
```suggestion
const CloseIconFC: React.FC = ({
```

You can omit props definition and just pass all props. `onClick` type will be applied by `styled` function